homeassistant/components/onewire/const.py

```diff
@@ -0,0 +1,12 @@
+"""Constants for 1-Wire component."""
+CONF_MOUNT_DIR = "mount_dir"
+CONF_NAMES = "names"
+
+DEFAULT_OWSERVER_PORT = 4304
+DEFAULT_SYSBUS_MOUNT_DIR = "/sys/bus/w1/devices/"
+
+DOMAIN = "onewire"
+
+SUPPORTED_PLATFORMS = [
```

Comment:
Where is this used?

Reply:
Linked to #39321 and follow-up PRs.

I have branches for new platforms and devices on my machine, and I was encouraged to use multiple smaller PRs for each instead of a big PR with all the new features.

Reply:
Small PRs are good. But we don't add code until it's used. 👍